config/nrfconnect/chip-module/Kconfig

```diff
@@ -66,3 +66,9 @@ config CHIP_DEBUG_SYMBOLS
 	default y
 	help
 	  Build the application with debug symbols.
+
+config CHIP_EXTENDED_DISCOVERY
```

Comment:
I think it could go to `zephyr/Kconfig` as it doesn't rely on nRF-specific features.